storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -0,0 +1,350 @@
+                          REMOTE_LOG_METADATA_MANAGER_CLASS_NAME_DOC)
+                  .define(REMOTE_LOG_METADATA_MANAGER_CLASS_PATH_PROP, STRING, null, MEDIUM,
+                          REMOTE_LOG_METADATA_MANAGER_CLASS_PATH_DOC)
+                  .define(REMOTE_LOG_METADATA_MANAGER_LISTENER_NAME_PROP, STRING, null, MEDIUM,
+                          REMOTE_LOG_METADATA_MANAGER_LISTENER_NAME_DOC)
+                  .define(REMOTE_LOG_INDEX_FILE_CACHE_TOTAL_SIZE_BYTES_PROP, LONG,
+                          DEFAULT_REMOTE_LOG_INDEX_FILE_CACHE_TOTAL_SIZE_BYTES, atLeast(1), LOW,
+                          REMOTE_LOG_INDEX_FILE_CACHE_TOTAL_SIZE_BYTES_DOC)
+                  .define(REMOTE_LOG_MANAGER_THREAD_POOL_SIZE_PROP, INT,
+                          DEFAULT_REMOTE_LOG_MANAGER_THREAD_POOL_SIZE, atLeast(1), MEDIUM,
+                          REMOTE_LOG_MANAGER_THREAD_POOL_SIZE_DOC)
+                  .define(REMOTE_LOG_MANAGER_TASK_INTERVAL_MS_PROP, LONG,
+                          DEFAULT_REMOTE_LOG_MANAGER_TASK_INTERVAL_MS, atLeast(1), LOW,
+                          REMOTE_LOG_MANAGER_TASK_INTERVAL_MS_DOC)
+                  .define(REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MS_PROP, LONG,
+                          DEFAULT_REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MS, atLeast(1), LOW,
+                          REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MS_DOC)
+                  .define(REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MAX_MS_PROP, LONG,
+                          DEFAULT_REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MAX_MS, atLeast(1), LOW,
+                          REMOTE_LOG_MANAGER_TASK_RETRY_BACK_OFF_MAX_MS_DOC)
+                  .define(REMOTE_LOG_MANAGER_TASK_RETRY_JITTER_PROP, DOUBLE,
+                          DEFAULT_REMOTE_LOG_MANAGER_TASK_RETRY_JITTER, between(0, 0.5), LOW,
+                          REMOTE_LOG_MANAGER_TASK_RETRY_JITTER_DOC)
+                  .define(REMOTE_LOG_READER_THREADS_PROP, INT, DEFAULT_REMOTE_LOG_READER_THREADS,
+                          atLeast(1), MEDIUM, REMOTE_LOG_READER_THREADS_DOC)
+                  .define(REMOTE_LOG_READER_MAX_PENDING_TASKS_PROP, INT,
+                          DEFAULT_REMOTE_LOG_READER_MAX_PENDING_TASKS, atLeast(1), MEDIUM,
+                          REMOTE_LOG_READER_MAX_PENDING_TASKS_DOC);
+    }
+
+    private final boolean enableRemoteStorageSystem;
+    private final String remoteStorageManagerClassName;
+    private final String remoteStorageManagerClassPath;
+    private final String remoteLogMetadataManagerClassName;
+    private final String remoteLogMetadataManagerClassPath;
+    private final long remoteLogIndexFileCacheTotalSizeBytes;
+    private final int remoteLogManagerThreadPoolSize;
+    private final long remoteLogManagerTaskIntervalMs;
+    private final long remoteLogManagerTaskRetryBackoffMs;
+    private final long remoteLogManagerTaskRetryBackoffMaxMs;
+    private final double remoteLogManagerTaskRetryJitterMs;
```

Comment:
Jitter is now a relative thing, not ms. So we need to name the variable appropriately.